### ISSUE TITLE
ansible-vault parameters order is wrong (maybe validation changed at some point)

### ISSUE THREAD

**Issue by silviuvulcan** [codeflows/vaulty#9]
Running F1 -> vaulty: decrypt and view Ansible Vault file I get the following error:

`usage: ansible-vault [-h] [--version] [-v] {create,decrypt,edit,view,encrypt,encrypt_string,rekey} ... ansible-vault: error: unrecognized arguments: --vault-password-file=/home/user/SOMEPATH/SOMEKEY --output=-`

by creating a simple wrapper around ansible-vault to dump the parameters it was called with, I can see that Vaulty uses the following call

````bash
ansible-vault --vault-password-file /home/user/SOMEPATH/SOMEKEY --output=- decrypt /home/user/repo/ansible-playbooks/vars/SOME_ENC_FILE.yml
````
which is not correct, it should be

````bash
ansible-vault decrypt --vault-password-file=/home/user/SOMEPATH/SOMEKEY --output=- /home/user/repo/ansible-playbooks/vars/SOME_ENC_FILE.yml
````

Probably it wasn't enforced before but ansible-vault specifies that the order is:

````bash
ansible-vault [-h] [--version] [-v]
                     {create,decrypt,edit,view,encrypt,encrypt_string,rekey}
                     ... <--- here go switches for the above chosen command
````
My ansible version is 2.9.9 running under WSL

**Comment by codeflows** [comment]
Thanks for the report! I'll fix this right away and hopefully will roll a new release soon.

**Comment by codeflows** [comment]
@silviuvulcan just published Vaulty 0.2.1 including this fix. If you can test it and report back to me that'd be great, thanks!

**Comment by silviuvulcan** [comment]
Hi @codeflows . Just tested 0.2.1 and is working great for me. Thanks for the quick fix !!
